tests/functional/test_cognito_idp.py

```diff
@@ -114,7 +111,5 @@ def run(self):
             operation(**self._parameters)
             request = self._http_stubber.requests[0]
 
-        assert_false(
-            'authorization' in request.headers,
+        assert 'authorization' not in request.headers, \
             'authorization header found in unsigned operation'
```

Comment:
We can probably make this a `failure_msg` param or something similar to avoid the line wrap.